### ISSUE TITLE
[Bug]:  Error with types

### ISSUE THREAD

**Issue by LopezSantiago** [sibiraj-s/ngx-editor#494]
### What happened?

The library has error with trusted-types/lib and prosemirror-view libraries
I created a new angular project and only add the library in latest version and crashed. I guess the problem is with the  "prosemirror-view": "1.31.6", I changed the version to 1.36.7 and added @types/trusted-types and the problem dissapeared.

### Version

16.0.0

### Angular Version

16.2.0

### Relevant log output

```shell
Error: node_modules/ngx-editor/lib/pipes/sanitize/sanitize-html.pipe.d.ts:3:29 - error TS2307: Cannot find module 'trusted-types/lib' or its corresponding type declarations.

3 import { TrustedHTML } from 'trusted-types/lib';
                              ~~~~~~~~~~~~~~~~~~~

Error: node_modules/ngx-editor/lib/trustedTypesUtil.d.ts:1:55 - error TS2307: Cannot find module 'trusted-types/lib' or its corresponding type declarations.    

Error: node_modules/ngx-editor/node_modules/prosemirror-view/dist/index.d.ts:215:9 - error TS2717: Subsequent property declarations must have the same type.  Property 'pmViewDesc' must be of type 'ViewDesc | undefined', but here has type 'ViewDesc | undefined'.

215         pmViewDesc?: ViewDesc;
            ~~~~~~~~~~

  node_modules/prosemirror-view/dist/index.d.ts:215:9
    215         pmViewDesc?: ViewDesc;
                ~~~~~~~~~~
    'pmViewDesc' was also declared here.
```

### Willing to submit a PR?

None

**Comment by LopezSantiago** [comment]
1.31.7

**Comment by sibiraj-s** [comment]
Okay. Regarding `trusted-types` It should be added to dependencies. The other is just something happens during upgrade often I guess, and it has nothing to do with the editor.

See here for steps to resolve it https://github.com/sibiraj-s/ngx-editor/issues/469#issuecomment-1636771997

**Comment by sibiraj-s** [comment]
Should be fixed in [v16.0.1](https://github.com/sibiraj-s/ngx-editor/releases/tag/v16.0.1)
